srcCpp/perftest_publisher.cxx

```diff
@@ -1367,18 +1367,25 @@ class LatencyListener : public IMessagingCB
             cpu.initialize();
         }
 
-        printf("Length: %5d  Latency: Ave %6.0lf us  Std %6.1lf us  "
-                "Min %6lu us  Max %6lu us  50%% %6lu us  90%% %6lu us"
-                " 99%% %6lu us  99.99%% %6lu us  99.9999%% %6lu us %s\n",
+        printf("Length: %5d"
+               " Latency: Ave %6.0lf " PERFT_TIME_UNIT
+               " Std %6.1lf " PERFT_TIME_UNIT
+               " Min %6lu " PERFT_TIME_UNIT
+               " Max %6lu " PERFT_TIME_UNIT
+               " 50%% %6lu " PERFT_TIME_UNIT
+               " 90%% %6lu " PERFT_TIME_UNIT
+               " 99%% %6lu " PERFT_TIME_UNIT
+               " 99.99%% %6lu " PERFT_TIME_UNIT
+               " 99.9999%% %6lu " PERFT_TIME_UNIT
```

Comment:
Probably we need to go for %6llu?

The maximum value on an unsigned long is 4294967295 that is equal to 4.29 seconds.

Reply:
Yes, most likely, I will add this as a different issue, as it involves refactoring the latency_ave, min, max, and history to be unsigned long longs

Reply:
#256